.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Documentation
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - "@typespec/**"
+  pull_request:
+    branches: ["main"]
+    paths:
+      - "@typespec/**"
+
+jobs:
+  push-JSON-schema:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install PNPM
+        uses: pnpm/action-setup@v4
+      - name: Set Up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: "package.json"
+          cache: "pnpm"
+      - name: Install Dependencies
+        run: pnpm install
+      - name: Compile Theme JSON Schema
```

Comment:
Compilation is not needed, only change tracking

Reply:
Ok!